### DISCUSSION TITLE
[BUG] Set-PnPTenantSite: Attempted to perform an unauthorized operation.

### DISCUSSION THREAD

**Discussion by mandrick91** [pnp/powershell#3897] [General]
I am Reporting an Issue with Set-PnPTenantSite

### Expected behavior 
I want to add second administrator (group or user) to a Site

### Actual behavior 
I get this error: Set-PnPTenantSite: Attempted to perform an unauthorized operation.

### Steps to reproduce behavior
 connect-pnponline -url "https://"CONTOSO".sharepoint.com/sites/"SITENAME"" -interactive
I login with Sharepoint Online admin role user or Global Admin
 set-pnptenantsite -Url "https://"CONTOSO".sharepoint.com/sites/"SITENAME"" -Owners ""USER or SEC. GROUP""
i always get this error: Set-PnPTenantSite: Attempted to perform an unauthorized operation.

### What is the version of the Cmdlet module you are running?
ModuleType Version    PreRelease Name                                                      
Manifest   2.4.0                 PnP.PowerShell                      
Manifest   2.2.0                 PnP.PowerShell                      
Manifest   2.1.1                 PnP.PowerShell                  

### Which operating system/environment are you running PnP PowerShell on?
- [X ] Windows 11
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

Nome              Id                        Versione     Origine
PowerShell 7-x64  Microsoft.PowerShell      7.4.2.0      winget
Terminale Windows Microsoft.WindowsTerminal 1.19.10821.0 winget

**Comment by AndersRask** [comment]
You need to log in as admin on the admin site to use Set-PnPTenantSite as mentioned in the documentation :)

Set-PnPTenantSite
SYNOPSIS
Required Permissions

SharePoint: Access to the SharePoint Tenant Administration site

https://pnp.github.io/powershell/cmdlets/Set-PnPTenantSite.html

`Connect-PnPOnline -Url "https://Contoso-admin.sharepoint.com" -Interactive`

**Reply by mandrick91** [reply]
Thank you so much, my bad error

**Reply by mandrick91** [reply]
But it doesn't works with some site, how could it be possible?
it seems to be some site created by Private or Shared Channel in Teams

**Reply by mandrick91** [reply]
Hi, @AndersRask
how can I work with that on some sub-site from Private or Shared Channel?
